sftp/1.3/modules/ROOT/edits/sftp-copy-move.adoc

```diff
@@ -0,0 +1,65 @@
+= Copy and Move Files with the SFTP Connector - Mule 4
+:page-aliases: connectors::sftp/sftp-copy-move.adoc
+
+Anypoint Connector for SFTP (SFTP Connector) Copy and Move operations enable you to copy and move files or directories on demand.
+
+While SFTP has a native concept of copying and moving, FTP does not. FTP Connector simulates these operations by downloading file content and writing it to a new location. Although all of this is done with concurrent streams that start writing and reading at the same time, copying and moving files does have a significant network overhead and should be used wisely. The SFTP Connector Move operation does not impact network overhead.
+
+== Configure the Copy Operation in Studio
+
+To add and configure the Copy operation in Studio, follow these steps:
+
+. In the *Mule Palette* view, search for `sftp` and select the *Copy* operation.
+. Drag the *Copy* operation onto the Studio canvas.
+. In the *General* tab of the operation configuration screen, click the plus sign (*+*) next to the *Connector configuration* field to access the global element configuration fields.
+. Specify the connection information and click *OK*.
+. Set the *Source path* field to the path of the file to copy, for example `source.txt`.
```

Comment:
comma
```suggestion
. Set the *Source path* field to the path of the file to copy, for example, `source.txt`.
```
You can always check this by looking up ", for example, " in the style ref.